_includes/icon.liquid

```diff
@@ -0,0 +1,27 @@
+{% assign addIcon = include.icon %}
+<img
+  {% if addIcon contains 'http' %}
```

Comment:
It is best to search for `://` since it could include files from other sources (e.g. ftp://)